tag14: [0.2580999476708011, -0.1868337771659926, -0.13957321012333898, 0.002370629880109964, -0.0019076751565287914, -0.7071951270330318, 0.7070118762723882]
tag13: [0.37382017307173643, -0.23473308980560093, 0.24011447946807993, 0.0017031343831912724, -0.0023948755849258966, -0.8082099835292995, 0.5888870744274957]
tag11: [0.41070065702158237, 0.3466906371064872, 0.5152584488575611, 0.5755061527516134, -0.4128010924874145, -0.41301568414025425, 0.5725434226680031]
tag12: [0.33532029687418763, -0.2747243603034431, 0.20512995395184447, -0.4084630084339034, 0.5606093231530197, 0.5826220512908117, -0.42358789275040093]
tag1: [0.47355947470631926, 0.06032451002403491, 0.4053034889562576, -0.500168823791138, 0.5013294006093921, 0.49993856179177315, -0.4985593387206685]
tag2: [0.47364349654923543, -0.05969611852366842, 0.4053014141190673, 0.5002434187949921, -0.4997980033881559, -0.4999130669185662, 0.5000454012243131]
tag4: [0.47370503958457777, -0.059724177807384426, 0.22526597728662037, 0.5010284853830083, -0.4989669598710235, -0.4997544780024592, 0.5002478300886135]
tag3: [0.4736984054721347, 0.06027399409515952, 0.22525912554437455, 0.5024411053444355, -0.4980512778741606, -0.4978462734416123, 0.5016442447476862]
tag10: [0.19026558047004577, 0.2603006290907498, -0.08805766448485283, 3.855154682522976e-06, 7.122312231087815e-05, 0.7070871416962369, 0.7071264165340244]